src/main/java/core/basesyntax/RandomColor.java

```diff
@@ -1,10 +1,13 @@
 package core.basesyntax;
 
-/**
- * <p>Имплементируйте необходимый функциональный интерфейс и его метод для случайного
- * выбора цвета</p>
- */
-public class RandomColor {
+import java.util.function.Supplier;
+
+public class RandomColor implements Supplier<String> {
     private static final String[] COLORS = {"red", "green", "blue", "yellow", "pink",
             "black", "white"};
+
+    @Override
+    public String get() {
+        return COLORS[(int) (Math.random() * 6 + 1)];
```

Comment:
Math.random returns value >=0 and < 1, so we can't get 0 element from COLORS using your formula.